website/members/views.py

```diff
@@ -170,7 +170,7 @@ class ProfileDetailView(DetailView):
     template_name = "members/user/profile.html"
 
     def setup(self, request, *args, **kwargs) -> None:
-        if "pk" not in kwargs:
+        if "pk" not in kwargs and request.member :
```

Comment:
```suggestion
        if "pk" not in kwargs and request.member:
```

With this space our black code style checking will fail, although it is not running somehow.

Reply:
Found out that the remote repository does not have github actions enabled, that is required for the tests to run.